NZBHydra2 fails to start corrupt 4.6.0

I've built a new docker container for Nzbhydra2 and it has begun to fail since 4.6.0

Brand new install no config results in this

```
Logging wrapper output to /config/logs/wrapper.log
2022-08-22 19:52:40,497  INFO - Determined java version as '11' from version string 'openjdk version "11.0.16.1" 2022-08-12'
2022-08-22 19:52:40,497  INFO - Starting NZBHydra main process with command line: java -Xmx256M -DfromWrapper -XX:TieredStopAtLevel=1 -noverify -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/config/logs -Dspring.output.ansi.enabled=ALWAYS -jar /app/lib/core-4.6.0-exec.jar --nobrowser --datafolder /config in folder /app
19:52:40.942 [main] DEBUG org.nzbhydra.config.ConfigReaderWriter - Copying YAML to /config/nzbhydra.yml
19:52:41.173 [main] DEBUG org.nzbhydra.config.ConfigReaderWriter - Using temporary file /config/nzbhydra.yml.bak
 in 'reader', line 2, column 1:
    ^
 in 'reader', line 336, column 3:
      ^
 at [Source: (File); line: 336, column: 3]
java.lang.RuntimeException: Config file /config/nzbhydra.yml corrupted. If you find a ZIP in your backup folder restore it from there. Otherwise you'll have to delete the file and start over. Please contact the developer when you have it running.
	at org.nzbhydra.NzbHydra.initializeAndValidateAndMigrateYamlFile(NzbHydra.java:247)
	at org.nzbhydra.NzbHydra.main(NzbHydra.java:119)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
Caused by: java.lang.RuntimeException: Config file /config/nzbhydra.yml corrupted. If you find a ZIP in your backup folder restore it from there. Otherwise you'll have to delete the file and start over. Please contact the developer when you have it running.
	at org.nzbhydra.NzbHydra.initializeAndValidateAndMigrateYamlFile(NzbHydra.java:247)
	at org.nzbhydra.NzbHydra.main(NzbHydra.java:119)
2022-08-22 19:52:41,196  ERROR - Main process shut down unexpectedly. If the wrapper was started in daemon mode you might not see the error output. Start Hydra manually with the same parameters in the same environment to see it
```

--- a/core/src/main/resources/config/baseConfig.yml
+++ b/core/src/main/resources/config/baseConfig.yml
@@ -332,7 +332,7 @@ notificationConfig:
   displayNotifications: true
   displayNotificationsMax: 5
 searching:
-    alwaysConvertIds: "NONE"
+  alwaysConvertIds: "NONE"
   alwaysShowQuickFilterButtons: false
   applyRestrictions: "NONE"
   coverSize: 128

